test-executor/README.md

```diff
@@ -0,0 +1,9 @@
+![Cognifide logo](http://cognifide.github.io/images/cognifide-logo.png)
+
+# Automated Exploratory Tests
+<p align="center">
+  <img src="https://github.com/Cognifide/aet/blob/master/misc/img/aet-logo-black.png?raw=true"
+         alt="AET Logo"/>
+</p>
+
+## Test-executor Module
```

Comment:
Could we add some description here?
I.e. 
Contains code responsible for processing uploaded `suite.xml` file and providing information for currently processed suites. (if this is correct).
(should have)